src/pocketmine/block/BlockIds.php

```diff
@@ -166,43 +166,44 @@
 	const CARROT_BLOCK = 141;
 	const POTATO_BLOCK = 142;
 	const WOODEN_BUTTON = 143;
-	const MOB_HEAD_BLOCK = 144, SKULL_BLOCK = 144;
-	const ANVIL = 145;
+	const HEAD_BLOCK = 144, MOB_HEAD_BLOCK = 144, SKULL_BLOCK = 144;
+	const ANVIL_BLOCK = 145, ANVIL = 145;
 	const TRAPPED_CHEST = 146;
 	const WEIGHTED_PRESSURE_PLATE_LIGHT = 147, LIGHT_WEIGHTED_PRESSURE_PLATE = 147, GOLD_PRESSURE_PLATE = 147;
 	const WEIGHTED_PRESSURE_PLATE_HEAVY = 148, HEAVY_WEIGHTED_PRESSURE_PLATE = 148, IRON_PRESSURE_PLATE = 148;
-	const COMPARATOR_BLOCK = 149, UNPOWERED_COMPARATOR_BLOCK = 149;
-	const POWERED_COMPARATOR_BLOCK = 150;
-	const DAYLIGHT_SENSOR = 151;
+	const COMPARATOR_BLOCK = 149, UNPOWERED_COMPARATOR_BLOCK = 149, UNPOWERED_COMPARATOR = 149;
+
+	const POWERED_COMPARATOR_BLOCK = 150, POWERED_COMPARATOR = 150;
+	const DAYLIGHT_SENSOR = 151, DAYLIGHT_DETECTOR = 151;
 	const REDSTONE_BLOCK = 152;
-	const NETHER_QUARTZ_ORE = 153;
+	const NETHER_QUARTZ_ORE = 153, QUARTZ_ORE = 153;
 	const HOPPER_BLOCK = 154;
 	const QUARTZ_BLOCK = 155;
 	const QUARTZ_STAIRS = 156;
 	const DOUBLE_WOOD_SLAB = 157, DOUBLE_WOODEN_SLAB = 157, DOUBLE_WOOD_SLABS = 157, DOUBLE_WOODEN_SLABS = 157;
 	const WOOD_SLAB = 158, WOODEN_SLAB = 158, WOOD_SLABS = 158, WOODEN_SLABS = 158;
 	const STAINED_CLAY = 159, STAINED_HARDENED_CLAY = 159;
-
-	const LEAVES2 = 161;
+	//const STAINED_GLASS_PANE = 160; //Confirmed
+	const LEAVES2 = 161, LEAVE2 = 161;
 	const WOOD2 = 162, TRUNK2 = 162, LOG2 = 162;
 	const ACACIA_WOOD_STAIRS = 163, ACACIA_WOODEN_STAIRS = 163;
 	const DARK_OAK_WOOD_STAIRS = 164, DARK_OAK_WOODEN_STAIRS = 164;
-	const SLIME_BLOCK = 165;
-
+	const SLIME_BLOCK = 165, const SLIMEBLOCK = 165;
 	const IRON_TRAPDOOR = 167;
-
+	const PRISMARINE = 168;
+	const SEA_LANTERN = 169;
 	const HAY_BALE = 170;
 	const CARPET = 171;
 	const HARDENED_CLAY = 172;
 	const COAL_BLOCK = 173;
 	const PACKED_ICE = 174;
 	const DOUBLE_PLANT = 175;
 
-	const INVERTED_DAYLIGHT_SENSOR = 178, DAYLIGHT_SENSOR_INVERTED = 178;
+	const INVERTED_DAYLIGHT_SENSOR = 178, DAYLIGHT_SENSOR_INVERTED = 178, DAYLIGHT_DETECTOR_INVERTED = 178;
 	const RED_SANDSTONE = 179;
 	const RED_SANDSTONE_STAIRS = 180;
-	const DOUBLE_RED_SANDSTONE_SLAB = 181;
-	const RED_SANDSTONE_SLAB = 182;
+	const DOUBLE_RED_SANDSTONE_SLAB = 181, DOUBLE_STONE_SLAB2 = 181;
```

Comment:
Why add a `STONE_SLAB_2` to replace `RED_SANDSTONE_SLAB`?? This is inaccurate and bad when we have a better way of naming it...